tests/test_auth.py

```diff
@@ -81,7 +82,7 @@ def test_client_bad_add(self):
         self.login()
         rv = self.client.post('/client',
                 data=dict(application_origins="bad data in"))
-        self.assertTrue("Invalid URL" in rv.data)
+        self.assertTrue("Invalid URL" in rv.data.decode('utf-8'))
```

Comment:
Can you rename the `rv` variable? 

@pbugni `rv` is a command to pdb, if you use it as a variable name it's difficult to evaluate inside pdb :)

Reply:
Can you use py.test-style `assert`?

Reply:
I think you can get a unicode string from the response using `get_data(as_text=True)`, so you don't need to explicitly decode the string as unicode

Reply:
What's a good name to rename rv to? Also, rv is in a handful of testing files - should I rename everywhere?

Reply:
And should I change everything I can to py.test-style `assert`?

Reply:
>What's a good name to rename rv to? Also, rv is in a handful of testing files - should I rename everywhere?

It's difficult to tell in this context, but `rv` is a http response object (try `type(rv)`), my goto is usually just `response`

>And should I change everything I can in this file to py.test-style assert?

Entirely up to you :). The focus here is py3 compliance, but I'm just taking the opportunity to improve the code you're already changing.